modules/wmco-prerequisites.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/understanding-windows-container-workloads.adoc
+
+[id="wmco-prerequisites_{context}"]
+= Windows Machine Config Operator (WMCO) prerequisites
+
+The following information details the supported cloud provider versions, Windows Server versions, and networking configurations. See the vSphere documentation for any information that is relevant to only that platform.
+
+== Supported Cloud Providers based on OCP and WMCO versions
```

Comment:
"cloud providers" in lowercase, they seem like ordinary nouns to me.

Reply:
So, on the second one: would the opening paragraph of the "Understanding topic cover the Windows Machine Config Operator (WMCO)" requirements and allow me to use the WMCO in the header?